ethereum/contracts/ETHApp.sol

```diff
@@ -181,4 +192,10 @@ contract ETHApp is RewardController, AccessControl {
         grantRole(INBOUND_CHANNEL_ROLE, _incentivized.inbound);
         emit Upgraded(msg.sender, c1, c2);
     }
+
+     function transferVaultOwnership(
+        address newOwner
+    ) external onlyRole(CHANNEL_UPGRADE_ROLE) {
+        vault.transferOwnership(newOwner);
```

Comment:
Should also emit an event, is useful for debugging.

Reply:
The `Ownable` contract emits `OwnershipTransferred` when the EtherVault is upgraded. Do we still need to add our own event to this contract?